source/browseMode.py

```diff
@@ -2108,8 +2108,8 @@ def _handleScrollTo(
 		"""Handle scrolling the browseMode document to a given object in response to an event.
 		Subclasses should call this from an event which indicates that the document has scrolled.
 		@postcondition: The virtual caret is moved to L{obj} and the buffer content for L{obj} is reported.
-		@param obj: The object to which the document should scroll.
-		@return: C{True} if the document was scrolled, C{False} if not.
+		:param obj: The object to which the document should scroll.
+		:return: C{True} if the document was scrolled, C{False} if not.
```

Comment:
Can you please remove the braces? I don't think they are valid RST/Sphinx

```suggestion
		:param obj: The object to which the document should scroll.
		:return: True if the document was scrolled, False if not.
```

Could you also fixup the precondition and note to use Sphinx if you are changing these lines